### PR TITLE
GLT-2233: Improved Arrays walkthrough for use in training

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: _includes/array_arrays.md [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-Note: Only Aliquot Samples may be added to an Array.
-{% endif %}
-
-# 3.1. Create an Array
+# 4.1. Create an Array
 
 1. On the _Arrays_ page, click the _Add_ button
-1. Choose an _Array Model_. If there are no options in this list, ask your MISO
-administrator to add some Array Models to MISO
+1. Choose _Array Model_ "Infinium MethylationEPIC-8 BeadChip.
```
Comment (callunity) [inline _includes/array_arrays.md]: missing close-quote marks

File: _includes/array_samples.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# 3. Samples
+
+Next, we will create the samples to be run in our array. {% if include.detailed == true %}Only Aliquot Samples can be
+added to arrays, so we will receive Aliquots directly in this walkthrough.{% endif %}
+
+## 3.1 Receiving Samples
+
+1. In the left hand menu under _Tracking_, click _Samples_.
+1. Click the _Create_ button at the top of the table.
+1. In the dialog that appears, select _Sample Class_: gDNA (aliquot), and _Quantity_: 8, and click _Create_.
```
Comment (callunity) [inline _includes/array_samples.md]: The `_Sample Class_` clause is only `if include.detailed == true`

File: _includes/array_samples.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    * _QC Status_: Ready
+    * _Purpose_: Extra
+{% else %}
+    * _QC Passed?_: True
+1. Enter a _Sample Alias_ in the form "PROJ_S#_#" for each sample. Replace "PROJ" with the short name of your project,
+   and replace the "#" symbol with numbers of your choosing.
+{% endif %}
+1. Enter "PROJ_1" for the _External Name_ of the first sample, replacing "PROJ" with your own project's shortname
+1. Enter "PROJ_2", "PROJ_3", etc... for the _External Name_ of each sample, incrementing by 1 each time so that they
+   each have a unique _External Name_.
```
Comment (callunity) [inline _includes/array_samples.md]: `External Name` blocks are only `if include.detailed == true` (so the Matrix Barcode instruction for plain sample can't reference it)

**Comment by djcooke** [comment]
@callunity: fixed

**Pull request merged**